apiml-security-common/test-programs/CheckPerm.java

```diff
@@ -52,13 +55,13 @@ public static void main(String args[]) {
         int okAccessLevel = Integer.parseInt(args[6]);
         int badAccessLevel = Integer.parseInt(args[7]);
 
-        checkPermission(userid, okClass, okResource, okAccessLevel, 0, 0, 0, 0);
-        checkPermission(userid, badClass, okResource, okAccessLevel, 143, 0x93800cf, -1, -1);  // Last two bytes are equal to PlatformErrno2.JRSAFResourceUndefined
-        checkPermission(userid, okClass, badResource, okAccessLevel, 143, 0x93800cf, -1, -1);  // Last two bytes are equal to PlatformErrno2.JRSAFResourceUndefined
-        checkPermission(userid, okClass, okResource, badAccessLevel, 139, 0x93800d9, -1, -1);  // Last two bytes are equal to PlatformErrno2.JRNoResourceAccess
-        checkPermission(badUserid, okClass, okResource, okAccessLevel, 139, 0x93800d9, 143, 0x93800f9);  // Last two bytes are equal to PlatformErrno2.JRNoResourceAccess (second pair of expected values is for RACF that returns JRSAFNoUser)
-        checkPermission(badUserid, okClass, badResource, okAccessLevel, 143, 0x93800cf, 143, 0x93800f9);  // Last two bytes are equal to PlatformErrno2.JRSAFResourceUndefined (second pair of expected values is for RACF that returns JRSAFNoUser)
-        checkPermission(badUserid, badClass, badResource, okAccessLevel, 143, 0x93800cf, 143, 0x93800f9);  // Last two bytes are equal to PlatformErrno2.JRSAFResourceUndefined (second pair of expected values is for RACF that returns JRSAFNoUser)
+        checkPermission(userid, okClass, okResource, okAccessLevel, 0, 0, 0, 0, 0, 0);
```

Comment:
Is there any reason why the numbers are just used as a numbers instead of as named constants? This is valid for the whole CheckPerm as a question.

Reply:
Named constants would be better. I will improve it.